src/sidebar/panel.js

```diff
@@ -118,7 +118,9 @@ ClassicEditor.create(document.querySelector('#editor'), {
             waitingToReconnect = false;
             clearTimeout(loginTimeout);
             // set title attr of footer to the currently logged in account
-            footerButtons.title = eventData.profile && eventData.profile.email;
+           
+            const footerButtonMail = "Sync to"
```

Comment:
This would need to be a new localization string.

Reply:
@Natim I tried doing the same. But it didn't work
```
 footerButtons.title =  `${browser.i18n.getMessage('localization_string')} ${eventData.profile && eventData.profile.email} `
```

Reply:
Should I add it for all localization files(languages) and do as above? Do tell me what needs to be done here

Reply:
Yes probably: `syncToMail=Sync to {mail}` and then: ${browser.i18n.getMessage('syncToMail', eventData.profile.email)}

Reply:
You would need to run: npm run postinstall to propagate your changes to the localization file.